sdk/src/main/java/io/dapr/client/ObjectSerializer.java

```diff
@@ -35,16 +38,29 @@ public class ObjectSerializer {
    */
   protected static final ObjectMapper OBJECT_MAPPER = new ObjectMapper()
```

Comment:
Could you please revert this, we don't want to change the default ObjectMapper instance.